sprokit/pipelines/image_processing/images_to_kwa.pipe

```diff
@@ -370,8 +370,8 @@ connect from detector.feature_set
 connect from extractor.descriptor_set
         to   matcher.descriptor_set
 
-connect from matcher.track_set
-        to   compute_homog.track_set
+connect from matcher.feature_track_set
```

Comment:
Please put these and other related changes on the separate PR for the new feature/object tracks.